.github/workflows/branch.yml

```diff
@@ -62,12 +60,18 @@ jobs:
     if: ${{ always() }}
     name: codacy-coverage-reporter
     steps:
-      - name: Download Coverage Report
-        uses: actions/download-artifact@v2
-        with:
-          name: code-coverage-report-${{ needs.coverage-artifact-name.outputs.ref }}
+      - uses: actions/checkout@v2
       - name: Run codacy-coverage-reporter
         uses: codacy/codacy-coverage-reporter-action@v1
         with:
           project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
           coverage-reports: coverage.xml
+      #- name: Download Coverage Report
```

Comment:
Remove commented out config?

Reply:
Honestly, I have been trying some different things to get codacy coverage to trigger and report correctly.

Currently, it's having trouble verifying the `CODACY_PROJECT_KEY`.

Reply:
Messaged privately to check the value you configured.